wpimath/src/main/native/include/frc/kinematics/SwerveDriveKinematics.h

```diff
@@ -76,6 +76,23 @@ class SwerveDriveKinematics {
         wpi::math::MathUsageId::kKinematics_SwerveDrive, 1);
   }
 
+  explicit SwerveDriveKinematics(
```

Comment:
Since swerve sim was removed, should this be moved to a separate PR?

Reply:
removed